cli/cmd/plugin/ui/web/tanzu-ui/src/shared/constants/defaults/azure.defaults.tsx

```diff
@@ -25,11 +25,130 @@ export const AZURE_DEFAULT_VALUES = {
     [AZURE_FIELDS.CLUSTER_SERVICE_CIDR]: '100.64.0.0/13',
     [AZURE_FIELDS.ACTIVATE_PROXY_SETTINGS]: false,
 };
+const SINGLE_NODE_FALL_BACK: Array<string> = [
```

Comment:
I love the simplicity of just using the ordered list of instance types instead of first checking the map of node profiles to instance type that this replaces. My recommendation would be renaming this to something like SINGLE_NODE_INSTANCE_TYPES_ORDERED or some name that describes how it is used a little better

Reply:
I would suggest: PREFERRED_SINGLE_NODE_INSTANCE_TYPES_ORDERED, because that distinguishes it from the actual AVAILABLE node instance types

Reply:
I like it